VRDR/DeathRecord.cs

```diff
@@ -1334,17 +1334,18 @@ public string CertificationRoleHelper
                         }
                         return ("Other");
                     }
+                    else
+                    {
+                        return code;
+                    }
                 }
                 return null;
             }
             set
             {
                 if ((value != null) && !VRDR.Mappings.CertifierTypes.FHIRToIJE.ContainsKey(value))
                 { //other
-                    if (DeathCertification == null)
-                    {
-                        CreateDeathCertification();
```

Comment:
why was this call removed?

Reply:
It is redundant with the code in on line 1299....